common/src/main/kotlin/com/alfresco/content/PermissionFragment.kt

```diff
@@ -48,6 +48,25 @@ open class PermissionFragment : Fragment() {
         }
     }
 
+    private fun requestOptionalPermissions(
+        permissions: List<String>,
+        continuation: CancellableContinuation<Boolean>
+    ) {
+        val denied = deniedPermissions(requireContext(), permissions)
+
+        println("PermissionFragment.requestOptionalPermissions $denied")
```

Comment:
Use `com.alfresco.Logger` if you want to keep the log, instead of `println`

Reply:
I already removed this.

Reply:
Done